core/src/main/java/com/xmartlabs/base/core/module/SessionInterceptor.java

```diff
@@ -13,12 +14,11 @@
 import okhttp3.Response;
 
 public class SessionInterceptor implements Interceptor {
-  @NonNull
-  private final AccessTokenProvider accessTokenProvider;
-
   @Inject
-  public SessionInterceptor(@NonNull AccessTokenProvider accessTokenProvider) {
-    this.accessTokenProvider = accessTokenProvider;
+  AccessTokenProvider accessTokenProvider;
+
+  public SessionInterceptor() {
```

Comment:
Nop.
If you use it you must add it to your app component (the module doesn't have one)